src/test/java/com/meilisearch/sdk/SearchRequestTest.java

```diff
@@ -44,9 +44,43 @@ void getQuery() {
         assertEquals("bubble", classToTest.getAttributesToRetrieve()[0]);
         assertEquals("highlight", classToTest.getAttributesToHighlight()[0]);
         assertEquals("crop", classToTest.getAttributesToCrop()[0]);
-        assertEquals("test='test'", classToTest.getFilter());
+        assertEquals(null, classToTest.getFilterArray());
+        assertEquals(1, classToTest.getFilter().length);
+        assertEquals("test='test'", classToTest.getFilter()[0]);
         assertEquals("facets", classToTest.getFacetsDistribution()[0]);
         assertEquals("sort", classToTest.getSort()[0]);
         assertEquals(900, classToTest.getCropLength());
+
+        classToTest =
+                new SearchRequest(
+                        "This is a Test",
+                        200,
+                        900,
+                        new String[] {"bubble"},
+                        new String[] {"crop"},
+                        900,
+                        new String[] {"highlight"},
+                        new String[][] {
+                            new String[] {"test='test'"}, new String[] {"test1='test1'"}
+                        },
+                        true,
+                        new String[] {"facets"},
+                        new String[] {"sort"});
+        assertEquals("This is a Test", classToTest.getQ());
+        assertEquals(200, classToTest.getOffset());
+        assertEquals(900, classToTest.getLimit());
+        assertEquals("bubble", classToTest.getAttributesToRetrieve()[0]);
+        assertEquals("highlight", classToTest.getAttributesToHighlight()[0]);
+        assertEquals("crop", classToTest.getAttributesToCrop()[0]);
+        assertEquals(null, classToTest.getFilter());
+        assertEquals(2, classToTest.getFilterArray().length);
+        assertEquals("test='test'", classToTest.getFilterArray()[0][0]);
+        assertEquals("test1='test1'", classToTest.getFilterArray()[1][0]);
+        assertEquals("facets", classToTest.getFacetsDistribution()[0]);
+        assertEquals("sort", classToTest.getSort()[0]);
+        assertEquals(900, classToTest.getCropLength());
+        assertEquals(
```

Comment:
I added a check with full built query, just to be safe.
The json I'm comparing with is 
`{
  "filter": [
    [
      "test='test'"
    ],
    [
      "test1='test1'"
    ]
  ],
  "q": "This is a Test",
  "attributesToRetrieve": [
    "bubble"
  ],
  "offset": 200,
  "facetsDistribution": [
    "facets"
  ],
  "limit": 900,
  "cropLength": 900,
  "attributesToHighlight": [
    "highlight"
  ],
  "sort": [
    "sort"
  ],
  "matches": true,
  "attributesToCrop": [
    "crop"
  ]
}`